README.md

```diff
@@ -85,6 +85,7 @@ As the API has just launched, not many implementations exist yet. If you are int
 | [ShivaHuang/swift-log-SwiftyBeaver](https://github.com/ShivaHuang/swift-log-SwiftyBeaver) | a logging backend for printing colored logging to Xcode console / file, or sending encrypted logging to [SwiftyBeaver](https://swiftybeaver.com) platform. |
 | [Apodini/swift-log-elk](https://github.com/Apodini/swift-log-elk) | a logging backend that formats, caches and sends log data to [elastic/logstash](https://github.com/elastic/logstash) |
 | [binaryscraping/swift-log-supabase](https://github.com/binaryscraping/swift-log-supabase) | a logging backend that sends log entries to [Supabase](https://github.com/supabase/supabase). |
+| [MahdiBM/DiscordBM](https://github.com/MahdiBM/DiscordBM) | a Discord package containing a standalone [DiscordLogger library](https://github.com/MahdiBM/DiscordBM/blob/discord-logger/README.md#discord-logger) |
```

Comment:
"a Discord package" sounds a bit weird, perhaps "a Discord client package"?

Reply:
Yeah it doesn't sound the best. I just wrote that based on what SPM calls them. The "DiscordLogger library" part also came from the fact that in Package.swift you do .library(name: "DiscordLogger"... couldn't also word that better.

Reply:
If you mean that it might sound like that Discord actually built the package, then i guess that's a valid point too.